Contributing.md

```diff
@@ -0,0 +1,50 @@
+# Stakeholders
+
+- **Project Manager**: responsible for aligning task creation with broader program direction through discussion with task manager(s). May comment on tasks.
+- **Task Manager**: responsible for day-to-day interfacing with github. Has authority to create, edit, and close tasks. Has authority to upload and remove files from the repository.
+- **Task Reviewer**: responsible for determining whether or not work submitted by assignee is complete. Has authority to close tasks after successful review. Has authority to upload files for completed tasks to the repository. May comment on tasks.
+- **Assignee**: the individual or working group assigned to and responsible for completing a task. May comment on tasks.
+# Task Format
+- Clearly defines the deliverable for the task.
+- As needed, clearly states metrics evaluating the quality and completeness of draft deliverables.
+- Is assigned value by the planning group.
+- Lists a deadline.
+- States dependencies, if any.
+
```

Comment:
In Scrum, stories (which is what they call tasks) are supposed to be formatted using the template
"as a <role> I want <feature>". Of all stories/tasks/issues I've read so far during my 20+ years working with software development, those are the only ones I've understood immediately what they were about even though they felt silly to phrase...
Please note that this template is more proper for the description. I've seen titles using this template, and that is unreadable...

Reply:
Ah like what's outlined [here?](https://www.mountaingoatsoftware.com/blog/advantages-of-the-as-a-user-i-want-user-story-template)